tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/NewUpgradePlanOp.java

```diff
@@ -57,6 +59,8 @@
 
 	public Value<String> getTargetVersion();
 
+	public Value<String> getUpgradeType();
```

Comment:
LEts call this `PlanName` or `getUpgradePlanName`

Reply:
hey Greg, I don't think planName and upgradePlanName are good choice because, the plan will have "name" by user.